### ISSUE TITLE
[EpicLoot 0.7.1] Bounty does not spawn in area

### ISSUE THREAD

**Issue by Larkenx** [RandyKnapp/ValheimMods#116]
![image](https://user-images.githubusercontent.com/12945799/113532152-aa51fe80-9598-11eb-8709-f71d7c55c0ff.png)

Searched for a couple of hours here...no enemy was found at all. I did notice it's right on top of a Burial Grounds. Maybe that has to do with it?

![image](https://user-images.githubusercontent.com/12945799/113532180-bd64ce80-9598-11eb-8ec4-b8da78436f68.png)

**Comment by RandyKnapp** [comment]
Thank you very much for the report, I'm trying to track down the cause of this too. Some questions:
1. Local Single-Player, Local Server, or Dedicated Server?
2. Did you look in the dungeon?

**Comment by Larkenx** [comment]
1. Local Single Player
2. I did fully explore the dungeon. I noticed some strange behavior though - the first time I went into the dungeon all the enemies were in a different collision box or something (I couldn't hit them or be hit). Quitting & restarting the game fixed the dungeon, but still no spawn for the bounty. I'm also wondering if a skeleton killed the bounty because when I arrived at the island on my boat, a few skeletons skirmished some Greydwarf/a Shaman. perhaps they killed it?

**Comment by Larkenx** [comment]
I do also have https://valheim.thunderstore.io/package/Smoothbrain/CreatureLevelAndLootControl/3.4.4/ installed; not sure if there is any incompatibilities at play there

**Comment by Joyg63tk** [comment]
I can confirm there is an issue with "desync" ? (areas and entities within them becoming un-interactive)
I have experienced this issue 2 times so far in 0.7.1
Both times the Bounty target didn't spawn but I encountered desync, other mobs would attack but deal no damage nor take any damage, no objects in the area where interactable.
Pulling mobs outside area allowed me to interact with them.
Logging out and back in fixed the desync however did not spawn the Bounty target.
The desync has also occurred outside of Bounty spawns when near a treasure chest, however this did not cause issues with the chest.
Dedicated server.
At the time of the issues I was the only person in the server.

This is all the information I can be sure of. my guess as someone who knows nothing is heavy load from all the mobs being handled by the client is causing the desync and the desync is preventing the Bounty target from spawning.
